cla_backend/apps/reports/api.py

```diff
@@ -13,6 +15,10 @@ class ExportListView(generics.ListAPIView, generics.DestroyAPIView):
     permission_classes = (IsAdminUser,)
     authentication_classes = (SessionAuthentication,)
     page_size = 1000
+    scheduled = False
 
     def get_queryset(self):
-        return super(ExportListView, self).get_queryset().filter(user=self.request.user)
+        if self.scheduled:
```

Comment:
Given these lines are otherwise the same, I wonder if it'd be easier on the eye to conditionally set `user` and then be able to just call the `super` in the same way regardless